src/operation/operation_model_interface.jl

```diff
@@ -15,12 +15,12 @@ get_name(model::OperationModel) = model.name
 get_store(model::OperationModel) = model.store
 is_synchronized(model::OperationModel) = is_synchronized(get_optimization_container(model))
 
-function get_requires_rebuild(model::OperationModel)
+function get_rebuild_model(model::OperationModel)
     sim_info = get_internal(model).simulation_info
     if sim_info === nothing
-        error("Model not part of a simulation")
+        error("Model not part of a simulation")ß
```

Comment:
Is the character at the end a typo?

Reply:
I have no idea how that character arrived there